docs: KB refresh from Zendesk tickets (Jun 1 – Jul 31, 2026) (#2995)
wandb/docs: support/models/articles/can-i-rename-a-team-or-entity.mdx

---

> title: "Can I rename a team or entity?"

Reviewer: I'm going back and forth: I don't like FAQs that are yes/no questions. I'd be tempted to rename this to "How do I rename..." even though you can't. Then the first two paras just need a little tweak which I'll suggest and I think would be good even if we keep to the yes/no framing.

Author: yeah, the debate is real. given that the answer is 'no', having it as 'can i' establishes clarity well.  'how do i ' seems the more likely question tho. i will rename the page.

keywords: ["Teams", "Team Management", "Org Management"]
---

No. W&B doesn't support renaming a team or entity. A team's name is set when you create the team and can't be changed afterward.

> To use a different name, create a new team with the name you want, then move your work into it:

Reviewer: ```suggestion
A team's name is set when you create the team and can't be changed afterward. Instead, create a new team with the name and members you want, then move your work into it:
```

> 1. Create the new team. See [How do I create a new team in W&B?](/support/models/articles/how-do-i-create-a-new-team-in-wandb).

Reviewer: I'm not sure the procedure belongs here at all. If you're creating a team, you probably need to be following the narrative docs, not this FAQ.

Author: well for actually creating the team, that i changed the link to source docs not KB.
But I think this procedure is still warranted, because the narritive in docs is for establishing a team,  and doesn't address the "moving of projects" that this FAQ specifically addresses to mimic a 'rename'.

> 2. Move your projects or runs into the new team. You can move runs between teams in the W&B App, but logged artifacts don't move automatically.

Reviewer: This could benefit from a link to how to move projects or runs.
```suggestion
2. Move your projects or runs into the new team. You can move runs between teams in W&B, but logged artifacts don't move automatically.
```
I don't think the App framing is useful here.

> 3. If you need an entire project moved, or you aren't able to create the team yourself, [contact support](mailto:support@wandb.com) with the source team, the destination team, and the projects to move. Support schedules the move during a window when nothing is actively logging to either team.

Reviewer: This seems weird. If you can't do these things, shouldn't you contact your own org admins?

Author: cleaned up the sentence to separate the priviledge statement from the product-capability issue. the tickets support that moving a project is a product gap, regardless of being an admin.

> Because the name is permanent, choose it carefully before you create the team.

Reviewer: Not sure this is needed again at this point.

<Note>

> A team's storage location is also fixed at creation. If you need a team whose data lives in your own cloud storage bucket, configure that bucket when you create the team. See [Secure storage connector](/platform/hosting/data-security/secure-storage-connector).

Reviewer: ```suggestion
A team's storage location is also fixed at creation. Configure a team's storage location when you create the team. See [Secure storage connector](/platform/hosting/data-security/secure-storage-connector).
```

Reviewer: But this could be moved up to the top, or maybe belongs in its own FAQ that we can link to. It's orthogonal.
